apps/common/utils/printer.py

```diff
@@ -0,0 +1,55 @@
+from termcolor import COLORS
+
+
+class ColoredPrinter(object):
+    _red = 'light_red'
+    _green = 'light_green'
+    _yellow = 'light_yellow'
+    _grey = 'light_grey'
+    _white = 'white'
+
+    @staticmethod
+    def polish(text, color):
+        fmt = u"\033[%sm%s\033[0m"
+        color_code = u'0;%s' % COLORS[color]
+        return u"\n".join([fmt % (color_code, t) for t in text.split(u'\n')])
+
+    @staticmethod
+    def _print(text):
+        print(text)
+
+    def red(self, text):
+        self._print(self.polish(text=text, color=self._red))
+
+    def green(self, text):
+        self._print(self.polish(text=text, color=self._green))
+
+    def yellow(self, text):
+        self._print(self.polish(text=text, color=self._yellow))
+
+    def info(self, text):
+        self._print(self.polish(text=text, color=self._grey))
+
+    def line(self, output='=', length=60):
+        text = '{}{}'.format('\n', output * length)
+        self._print(self.polish(text=text, color=self._white))
+
+
+class ColoredFilePrinter(ColoredPrinter):
+    def __init__(self, file, flush_now=False):
+        self.file = file
+        self.flush_now = flush_now
+
+    def _print(self, text):
+        self.file.write(text + '\r\n')
+        if self.flush_now:
+            self.file.flush()
+
+    def close(self):
+        try:
+            self.file.close()
+        except Exception: # noqa
+            pass
+
+    def __del__(self):
+        self.close()
```

Comment:
I have reviewed the given code and found no significant issues or optimizations to be made at this moment.

However, here are some minor suggestions:

1. In `ColoredFilePrinter`, I think you might not need to check for exceptions when calling `close()`. It's fine to just do it explicitly like so:
   ```python
       def close(self):
           self.file.close()
   ```
   
2. You could consider adding type hints to improve readability of the code:
   ```python
       class ColoredPrinter(object):  
          ...
   ```
   
   Or even better, use annotated functions which can make typing easier.
     
The general review suggests that apart from these recommendations, there is nothing out-of-place in terms the current state of this code. The changes will further ensure clearness while maintaining efficiency & simplicity.